float32 pressure
float32 temperature
float32 depth
float32 altitude
float32 shuntvoltage
float32 busvoltage
float32 current_mA
float32 power_mW
float32 loadvoltage